### ISSUE TITLE
create sorting algorithms

### ISSUE THREAD

**Issue by EnzoRobaina** [ambujraj/hacktoberfest2018#117]
Added this issue to be referenced when adding sorting algorithms.

**Comment by EnzoRobaina** [comment]
> I would like to take up this issue.

There are many languages to work on, be sure to check current algorithms before starting a new pull request.

**Comment by jcert** [comment]
Can I add a heapsort in c++?

**Comment by GOVINDDIXIT** [comment]
@ambujraj pls review the PR

**Comment by anshulkhattar** [comment]
Added heap sort via #914

**Comment by brunabazaluk** [comment]
doing pancake sorting in c

**Comment by dhruv997** [comment]
Done Tim_Sort using cpp #1101

**Comment by shubhamkakkar** [comment]
I am making quickSort in javascript

**Comment by saurabh28march** [comment]
I will add BFS Algorithm using STL
